nemo/collections/nlp/modules/common/token_classifier.py

```diff
@@ -70,6 +70,88 @@ def __init__(
             use_transformer_init: whether to initialize the weights of the classifier head with the same approach used in Transformer
         """
         super().__init__()
+        self.mlp = MultiLayerPerceptron(
+            hidden_size, num_classes, num_layers=num_layers, activation=activation, log_softmax=log_softmax
+        )
+        self.dropout = nn.Dropout(dropout)
+        if use_transformer_init:
+            self.apply(lambda module: transformer_weights_init(module, xavier=False))
+
+    @typecheck()
```

Comment:
could you please check default values for TokenClassifier? num_layers is 2 as far as I remember, smth else could be also different, these defaults are from BERTTokenClassifier

Reply:
done